deputy_project_leads.md

```diff
@@ -38,6 +38,20 @@ This include checking that:
   the existing functions
 
 
+## Principle Engineer
```

Comment:
The correct spelling is *principal*.  See e.g. https://www.dictionary.com/e/principal-vs-principle/ :)